cmd/adapter-rest/startcmd/start.go

```diff
@@ -265,18 +272,19 @@ type adapterRestParameters struct {
 	staticFiles                 string
 	presentationDefinitionsFile string
 	// TODO assuming same base path for all hydra endpoints for now
-	hydraURL             string
-	mode                 string
-	didCommParameters    *didCommParameters // didcomm
-	trustblocDomain      string
-	universalResolverURL string
-	governanceVCSURL     string
-	requestTokens        map[string]string
-	walletAppURL         string
-	oidcClientDBKeyPath  string
-	externalURL          string
-	didAnchorOrigin      string
-	contextProviderURLs  []string
+	hydraURL              string
+	mode                  string
+	didCommParameters     *didCommParameters // didcomm
+	trustblocDomain       string
+	universalResolverURL  string
+	governanceVCSURL      string
+	requestTokens         map[string]string
+	walletAppURL          string
+	oidcClientDBKeyPath   string
+	externalURL           string
+	didAnchorOrigin       string
+	contextProviderURLs   []string
+	outputDescriptorsFile string
```

Comment:
I think this should be called `outputDescriptorsFilePath` just to make it clear that it's just the path to the file, not the actual file itself